### ISSUE TITLE
Toastify: Notification Color Contrast

### ISSUE THREAD

**Issue by bpmcneilly** [benetech/MathShare#956]
The toastify components are used to present users with notifications. The colors for various states have been updated from the toastify defaults, but these new colors (with the exception of the error content) are below the 4.5:1 required for normal sized text.

The text for all toastify notifications are presented in white (#ffffff). The background color for warnings (`class=“Toastify__toast--warning”`) is an orange (#f89406) which has a color contrast ratio of 2.28:1. Success messages (`class=“Toastify__toast--success”`) have a light green (#51a351) background which has a color contrast ratio of 3.13:1. Finally, generic info notes (`class=“Toastify__toast--info”`) have a teal (#2f96b4) background which has a color contrast ratio of 3.42:1.

#### Suggested Solution
Each of the listed background colors would have a sufficient color contrast ratio (in every case at least 6.0:1) if their text color were changed to black (#000000). However, if keeping the white text is preferred, each background color could be darkened until a sufficient color contrast ratio is attained.

**Comment by johnhbenetech** [comment]
Hey @alexrcabral @benetechMatt please suggest an alternate color scheme for these buttons that meets the contrast required.

If I don't receive a suggestion by Monday, I will make a decision and move forward

**Comment by johnhbenetech** [comment]
Let's do the following:

#CD0E15 - warning
#3D7B3D - success
#267992 - generic
